### PR TITLE
Fix/tabs accessibility

### PR DESCRIPTION
**Pull request by odero**
For reviewers: click the individual commits for review. You can skip the "Fix spacing and indentation commit" - it's why I had it as a separate commit

Fixes
- tab dropdowns - fixed aira -> aria typos; added `aria-haspopup="true" aria-expanded="false"` for the reader
- removed `tabindex=-1` since it was resulting in incorrect navigation when hitting tab to move across elements

### CODE REVIEW

**Comment by gathogojr** [comment]
@odero This pull request is extremely hard to review. It's hard to pick out the material changes in the diff. I think the html formatting changes should have been made in a different pull request.

**Comment by wandeg** [comment]
Do you have before and after screenshots?

File: pages/getting-started/understand-odata-in-6-steps.html [hunk 1]
```diff
@@ -6,1413 +6,1677 @@
...
+    <div class="modal-content">
+      <div class="modal-header">
+        <button type="button" class="close" data-dismiss="modal"><span aria-hidden="true">&times;</span><span
+            class="sr-only">Close</span></button></p>
+        <h4 class="modal-title" id="step-collection-modal-label">Response</h4>
+        </p>
+      </div>
+      <div class="modal-body">
+        {% highlight js %}
+        HTTP/1.1 200 OK
```
Comment (habbes) [inline pages/getting-started/understand-odata-in-6-steps.html:98]: Will this leading indentation also appear on the rendered page? Or will `highlight` take care of that?

File: pages/getting-started/understand-odata-in-6-steps.html [hunk 2]
```diff
@@ -6,1413 +6,1677 @@
...
+        Content-Type: application/json; odata.metadata=minimal
+        OData-Version: 4.0
+        {
+        '@odata.context':
+        'https://services.odata.org/V4/(S(a2k31bgwiyejn2j2iiybvq4p))/TripPinServiceRW/$metadata#People',
+        '@odata.nextLink':
+        'https://services.odata.org/V4/(S(a2k31bgwiyejn2j2iiybvq4p))/TripPinServiceRW/People?%24skiptoken=8',
+        'value': [
+        {
+        '@odata.id':
```
Comment (habbes) [inline pages/getting-started/understand-odata-in-6-steps.html:109]: Should this be indented as well?

**Comment by habbes** [comment]
Could you update the description with a summary of the issue, the changes made and how they fix the issue?

**Comment by odero** [comment]
@gathogojr, @habbes - see the PR description - 
@wandeg - it's a screen-reader fix, can't capture a screenshot for that

File: pages/getting-started/understand-odata-in-6-steps.html (hunk 1 above)
Comment (odero) [reply pages/getting-started/understand-odata-in-6-steps.html:98]: Nice catch

**Review by gathogojr — APPROVED** [review]
:shipit:

**Review by marabooy — APPROVED** [review]

**Comment by marabooy** [comment]
Could you confirm if the js code in line 1366 needs to be updated to not trap user focus using tab index or probably works great now

**Review by habbes — APPROVED** [review]

**Pull request merged**